drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c

```diff
@@ -311,7 +311,7 @@ static int nrf_wifi_util_tx_stats(const struct shell *shell,
 				  size_t argc,
 				  const char *argv[])
 {
-	struct vif_ctx_zep *vif_ctx_zep = NULL;
+	struct wifi_nrf_vif_ctx_zep *vif_ctx_zep = NULL;
```

Comment:
Is this even used? It seems that this is only assigned to but not used (which is another warning).